_posts/2021-04-07-cultural-competency-in-archives-phase-one-of-the-education-campaign.md

```diff
@@ -0,0 +1,52 @@
+- Did any of these courses make you think differently about an experience at the RAC? In the reading room? Reading through or writing archival description? Interacting with donors?
+- What activities, discussion questions, and/or exercises presented within the learning modules did you find the most engaging and/or challenging?
+
+## Discussion Sessions
+
+While Amy, Darren, and I organized and facilitated this phase of the education campaign, it is important to note that we were also learning.  We took the courses, answered the pre-survey questions, and shared our thoughts as part of the sessions. Our experience with the learning modules guided the way we structured the sessions in ways that encouraged an inclusive and open dialogue.  At the start of each session, we provided [ground rules](https://extension.umn.edu/public-engagement-strategies/setting-ground-rules-productive-discussions) based on guidance from the University of Minnesota, for example: seek first to understand, not to be understood, and criticize the discussion, not the person. Prior to the session,  we asked that everyone prepare an answer to an icebreaker question: “What was your biggest takeaway from the learning modules?” We wanted to provide everyone the opportunity to express their views on the modules and ensure that all voices were heard.
+
+After the initial icebreaker, we moved into breakout rooms for a larger conversation about the Rockefeller Archive Center itself.  Because we encouraged all staff to participate, we understood that there might be discomfort expressing certain views in front of higher-ups and teammates. To encourage more free discussion, we created breakout rooms groups that varied across the hierarchy of the organization and across departments. The main discussion consisted of three broad topics.
+
+*Topics for Discussion*
+
+- Workplace Culture at the  RAC - reactions to answers from the pre-survey
+- RAC as an Institution - How can we take first impressions into account when greeting researchers? How we might contribute to [gatekeeping](https://blogs.ifla.org/faife/2019/07/17/from-gatekeeper-to-gateway-to-gate-opener-the-changing-role-of-libraries-and-how-we-talk-about-it/)?
+- Whiteness at the Rockefeller Archive Center - How might the demographic makeup of our records’ creators impact the way researchers, staff members, and others perceive the RAC? How may the ways in which the creators perceive and describe the subjects differ from the ways in which the subjects perceive and describe themselves?
+
+In total, the discussion sessions were scheduled for an hour and a half each. To manage time, we tried to keep the icebreaker around 20 minutes with 45 minutes left for the three discussion sections and 10 minutes at the end to report back from breakout rooms. Following the sessions, we sent out a post-survey to inform how we move forward into a monthly reading group for phase two of the education campaign.
```

Comment:
like the pre-survey, it may be useful to note that the post-survey was also anonymous.